coordinator/src/routes.rs

```diff
@@ -112,9 +117,26 @@ pub async fn post_fake_scid(
         ))
     })?;
 
-    Ok(Json(
-        app_state.node.inner.create_intercept_scid(target_node),
-    ))
+    let (fscid, _) = app_state.node.inner.create_intercept_scid(target_node);
+    Ok(Json(fscid))
+}
+
+pub async fn post_fake_scid_with_fee_hint(
```

Comment:
:wrench: I would either split these two, so that you can just introduce a new API that only fetches the fee hint, or rename the API and the route to reflect that it's not just the fake SCID anymore.

We could go for `register_interceptable_invoice` or something like that to connect it to the fact that the call is used to inform the coordinator that we are gonna route an interceptable payment through it (with the fake SCID and fee that it gives us).